### PR TITLE
[KOGITO-162] - Migrate the Kogito base images from centos to ubi8

### PR DESCRIPTION
**Pull request by spolti**
Signed-off-by: Filippe Spolti <fspolti@redhat.com>

### CODE REVIEW

**Comment by spolti** [comment]
@mswiderski @ricardozanini fyi

**Review by mswiderski — APPROVED** [review]

**Review by ricardozanini — APPROVED** [review]

File: onboarding-example/hr/readme.md [hunk 1]
```diff
@@ -76,14 +76,14 @@ NOTE: Make sure that kogito S2I image builders are available to your OpenShift e
...
-oc new-build myproject/kogito-quarkus-centos-s2i --binary=true --name=hr-service-builder
+oc new-build myproject/kogito-quarkus-ubi8-s2i --binary=true --name=hr-service-builder
 oc start-build hr-service-builder --from-dir . --incremental=true
 ```
 
 Once the build is completed create new build for runtime image
 
 ```sh
-oc new-build --name hr-service --source-image=hr-service-builder --source-image-path=/home/kogito/bin:. --image-stream=kogito-quarkus-centos
+oc new-build --name hr-service --source-image=hr-service-builder --source-image-path=/home/kogito/bin:. --image-stream=kogito-quarkus-ubi8
```
Comment (ricardozanini) [inline onboarding-example/hr/readme.md:86]: We still need the `source-image-path` parameter?

File: onboarding-example/onboarding/readme.md [hunk 2]
```diff
@@ -78,14 +78,14 @@ NOTE: Make sure that kogito S2I image builders are available to your OpenShift e
...
-oc new-build myproject/kogito-quarkus-centos-s2i --binary=true --name=onboarding-service-builder
+oc new-build myproject/kogito-quarkus-ubi8-s2i --binary=true --name=onboarding-service-builder
 oc start-build onboarding-service-builder --from-dir . --incremental=true
 ```
 
 Once the build is completed create new build for runtime image
 
 ```sh
-oc new-build --name onboarding-service --source-image=onboarding-service-builder --source-image-path=/home/kogito/bin:. --image-stream=kogito-quarkus-centos
+oc new-build --name onboarding-service --source-image=onboarding-service-builder --source-image-path=/home/kogito/bin:. --image-stream=kogito-quarkus-ubi8
```
Comment (ricardozanini) [inline onboarding-example/onboarding/readme.md:88]: Same here

File: onboarding-example/payroll/readme.md [hunk 3]
```diff
@@ -80,14 +80,14 @@ NOTE: Make sure that kogito S2I image builders are available to your OpenShift e
...
-oc new-build myproject/kogito-quarkus-centos-s2i --binary=true --name=payroll-service-builder
+oc new-build myproject/kogito-quarkus-ubi8-s2i --binary=true --name=payroll-service-builder
 oc start-build payroll-service-builder --from-dir . --incremental=true
 ```
 
 Once the build is completed create new build for runtime image
 
 ```sh
-oc new-build --name payroll-service --source-image=payroll-service-builder --source-image-path=/home/kogito/bin:. --image-stream=kogito-quarkus-centos
+oc new-build --name payroll-service --source-image=payroll-service-builder --source-image-path=/home/kogito/bin:. --image-stream=kogito-quarkus-ubi8
```
Comment (ricardozanini) [inline onboarding-example/payroll/readme.md:90]: and here :)

File: openshift-example-tests/README.md [hunk 4]
```diff
@@ -12,10 +12,10 @@ To properly configure OpenShift and related images user needs to specify these p
...
-* image.kaas.quarkus.runtime - Tag for Quarkus runtime image, for example built from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-quarkus-centos
-* image.kaas.springboot.builder.s2i - Tag for S2I image, for example buil from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-springboot-centos-s2i
-* image.kaas.springboot.runtime - Tag for runtime image, for example built from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-springboot-centos
+* image.kaas.quarkus.builder.s2i - Tag for S2I Quarkus image, for example built from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-quarkus-ubi8-s2i
+* image.kaas.quarkus.runtime - Tag for Quarkus runtime image, for example built from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-quarkus-ubi8
+* image.kaas.springboot.builder.s2i - Tag for S2I image, for example buil from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-springboot-ubi8-s2i
+* image.kaas.springboot.runtime - Tag for runtime image, for example built from https://github.com/kiegroup/kogito-cloud/tree/master/s2i/kaas-springboot-ubi8
```
Comment (ricardozanini) [inline openshift-example-tests/README.md]: `kogito` or `kaas` prefix?

**Review by ricardozanini — CHANGES_REQUESTED** [review]

File: openshift-example-tests/README.md (hunk 4 above)
Comment (mswiderski) [reply openshift-example-tests/README.md]: Kogito makes more sense I believe

**Comment by mswiderski** [comment]
@spolti @ricardozanini shall this be included in 0.3.0 release? I am about to start the release in few hours...

**Comment by spolti** [comment]
@mswiderski @ricardozanini that's your call, let me update it to address the Ricardo's review. (Sorry for the delay, crazy day today)

**Comment by ricardozanini** [comment]
@spolti @mswiderski  +1 for 0.3.0

File: openshift-example-tests/README.md (hunk 4 above)
Comment (spolti) [reply openshift-example-tests/README.md]: I missed this one :)

**Comment by mswiderski** [comment]
@spolti @ricardozanini has this been already updated?

File: onboarding-example/hr/readme.md (hunk 1 above)
Comment (spolti) [reply onboarding-example/hr/readme.md:86]: yes, on ocp it is needed:
error: --source-image-path must be specified when --source-image is specified.

**Comment by ricardozanini** [comment]
@mswiderski I guess @spolti was testing the `source-image-path` parameter

File: openshift-example-tests/README.md (hunk 4 above)
Comment (spolti) [reply openshift-example-tests/README.md]: It will requires changes on some java code, do we still have time for it?
@mswiderski @ricardozanini ?

Comment (ricardozanini) [reply openshift-example-tests/README.md]: @spolti I don't think so, we could track this in another JIRA.

Comment (mswiderski) [reply openshift-example-tests/README.md]: I'd say let's skip it for now, not urgent/crucial for the release, maybe create jira and fix it for next release

Comment (spolti) [reply openshift-example-tests/README.md]: Alright, so it is ready :)

**Pull request merged**